getting_started/first_2d_game/06.heads_up_display.rst

```diff
@@ -48,7 +48,7 @@ well. There is a font file included in the game assets called
 Once you've done this on the ``ScoreLabel``, you can click the down arrow next
 to the Font property and choose "Copy", then "Paste" it in the same place
 on the other two Control nodes.
-Set "Custom Font Size" property of the ``ScoreLabel``. A setting of ``64`` works well.
+Set "Font Size" property of the ``ScoreLabel``. A setting of ``64`` works well.
```

Comment:
Shouldn't it also refer to the "Theme Overrides" section?

```suggestion
Set the "Font Size" property of the ``ScoreLabel`` under "Theme Overrides > Font Sizes". A setting of ``64`` works well.
```

Reply:
I had gone back and forth on whether to style it that way but I agree with the suggestion. It makes it more explicit and clear.